lib/src/test/java/com/basistheory/android/service/ProxyApiTests.kt

```diff
@@ -0,0 +1,192 @@
+package com.basistheory.android.service
+
+import com.basistheory.ApiClient
+import com.basistheory.ApiResponse
+import com.basistheory.android.model.ElementValueReference
+import io.mockk.every
+import io.mockk.impl.annotations.SpyK
+import io.mockk.junit4.MockKRule
+import io.mockk.slot
+import io.mockk.spyk
+import io.mockk.verify
+import junitparams.JUnitParamsRunner
+import junitparams.Parameters
+import kotlinx.coroutines.runBlocking
+import okhttp3.Call
+import okio.Buffer
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import strikt.api.expectThat
+import strikt.assertions.isA
+import strikt.assertions.isEqualTo
+import strikt.assertions.isNull
+import strikt.assertions.isTrue
+import java.util.*
+
+@RunWith(JUnitParamsRunner::class)
+class ProxyApiTests {
```

Comment:
All of these tests are failing - the mock doesn't seem to be intercepting the request